do-in-podman.sh

```diff
@@ -5,9 +5,10 @@ IMAGE=ghcr.io/ddterm/ci-docker-image
 IMAGE_VERSION=2023.07.04.0
 
 SCRIPT_DIR=$(CDPATH="" cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+HOME_DIR="${SCRIPT_DIR}/.container-home"
 
 TTY_FLAG=$(test -t 0 && echo -n -t)
 
 set -ex
 
-exec podman run --init --rm -i $TTY_FLAG -v "${SCRIPT_DIR}:${SCRIPT_DIR}" -w "${PWD}" "${IMAGE}:${IMAGE_VERSION}" xvfb-run "$@"
+exec podman run --init --rm -i $TTY_FLAG -e HOME=${HOME_DIR} -v "${SCRIPT_DIR}:${SCRIPT_DIR}:z" -w "${PWD}" "${IMAGE}:${IMAGE_VERSION}" xvfb-run "$@"
```

Comment:
Please add quotes around `HOME=${HOME_DIR}`. It could contain spaces.

Instead of relabeling, please do `--security-opt label=disable`.